docs/plugins/markdown/revealjs/README.md

```diff
@@ -0,0 +1,155 @@
+# revealjs
+
+<NpmBadge package="@vuepress/plugin-revealjs" />
+
+Add presentation in your VuePress site via Reveal.js.
+
+<!-- more -->
+
+## Usage
+
+```bash
+npm i -D @vuepress/plugin-prismjs@next
+```
+
+```js {7} title=".vuepress/config.js"
+import { revealJsPlugin } from '@vuepress/plugin-revealjs'
+
+export default {
+  plugins: [
+    revealJsPlugin({
+      // Enable reveal.js
+      revealJs: true,
+    }),
+  ],
+}
+```
+
+## Slide Syntax
+
+- Use `---` to split slides
+- Use `--` to split the slides second time (vertical display)
+
+```md
+@slidestart
+
+<!-- slide1 -->
+
+---
+
+<!-- slide2 -->
+
+---
+
+<!-- slide3 -->
+
+@slideend
+```
+
+By default, we use `auto` theme to render the presentation, but you can also use other themes with `@slidestart THEME_NAME`.
+
+You can enable the following themes in reveal.js via `themes` in plugin options:
+
+- `auto` (Default)
+- `black`
+- `white`
+- `league`
+- `beige`
+- `sky`
+- `night`
+- `serif`
+- `simple`
+- `solarized`
+- `blood`
+- `moon`
+
+For the appearance of each theme, see [Themes demo](themes.md).
+
+## Slide Layout
+
+By default, the plugin registers a global component `SlidePage` to render slides. In pages using this layout, you should only include a single slide syntax and no other contents to avoid rendering problems.
+
+If you do not need this layout or want to use another name, customize `layout` in plugin options.
```

Comment:
As a developer, I can understand the description here, but for users, a sample containing `frontmatter.layout` is needed.
```md
---
layout: SlidePage
---
@slidestart
<!-- content -->
@slideend
```